declarations_site/catalog/templates/results.jinja

```diff
@@ -16,31 +18,199 @@
 <section id="page">
     <div class="container">
         <div id="breadcrumbs">
-            <ol class="breadcrumb">
+            <ol class="breadcrumb pull-left">
                 <li><a href="{{ url("wagtail_serve", "") }}">Головна</a></li>
                 <li class="active">Результати пошуку</li>
             </ol>
             {{ social("Результати пошуку по: %s. " % query,
-                              "http://declarations.com.ua%s" % request.path ,
-                              add_disqus=False) }}
+                              "http://declarations.com.ua%s" % request.get_full_path() ,
+                              add_disqus=False,
+                              add_rss=True) }}
         </div>
 
-        <h1 id="page-header"><span>{{ query|default("Всі", true)|escape }} ({{ results.paginator.count }})</span>
+        <div id="ex-search-form">
```

Comment:
You have a huge chunk of code (which is basically search form) sitting in the template of search results. My proposition here is to move that form completely to the include or macro